cf/data/data.py

```diff
@@ -8746,22 +8747,21 @@ def floor(self, inplace=False, i=False):
         d._set_dask(da.floor(dx), reset_mask_hardness=False)
         return d
 
+    @daskified(_DASKIFIED_VERBOSE)
+    @_inplace_enabled(default=False)
     @_deprecated_kwarg_check("i")
-    def outerproduct(self, e, inplace=False, i=False):
+    def outerproduct(self, a, inplace=False, i=False):
         """Compute the outer product with another data array.
 
         The axes of result will be the combined axes of the two input
-        arrays:
+        arrays.
 
-          >>> d.outerproduct(e).ndim == d.ndim + e.ndim
-          True
-          >>> d.outerproduct(e).shape == d.shape + e.shape
-          True
+        .. seealso :: `np.multiply.outer`
```

Comment:
```suggestion
        .. seealso:: `np.multiply.outer`
```

(else I don't think the Sphinx admonition will be recognised and render properly).